src/store/modules/nft.js

```diff
@@ -134,6 +143,7 @@ const getters = {
   getNFTClassListingInfoById: state => id => state.listingInfoByClassIdMap[id],
   getNFTClassMetadataById: state => id => state.metadataByClassIdMap[id],
   getNFTClassOwnerInfoById: state => id => state.ownerInfoByClassIdMap[id],
+  getNFTClassPriceInfoById: state => id => state.priceInfoByClassIdMap[id],
```

Comment:
potentially confusing that this function only contains `listing` and `fiat`, but not price info from wnft...

Reply:
Updated